test/torch/tensors/test_large_precision.py

```diff
@@ -218,3 +218,12 @@ def test_mod():
     lpt2 = x2.fix_prec(internal_type=internal_type, precision_fractional=precision_fractional)
     result = lpt1 % lpt2
     assert torch.all(torch.eq(expected, result.float_precision()))
+
+
+def test_long():
```

Comment:
can you parametrize this test and use multiple more input + expected values? Like an array with more than one value, float numbers with more decimal places (example 1.234123), negative numbers...